### DISCUSSION TITLE
[Question][Hw5] Getting cond of icmp

### DISCUSSION THREAD

**Discussion by ee12ha0220** [prosyslab-classroom/cs348-information-security#604] [Q&A]
The icmp instruction in Llvm IR looks like this. 

icmp (cond) (ty) (op1), (op2)

Is there a way I can get information about (cond) in the type of  Llvm.Icmp.t?

**Comment by Altterisk** [comment]
I used Llvm.icmp_predicate to get the Llvm.Icmp.t

**Comment by ee12ha0220** [comment]
Thanks a lot!
